src/components/FontChangeButton.scss

```diff
@@ -4,25 +4,176 @@
+//   @-moz-keyframes fade-in-show{0% {opacity:0; -moz-transform:translateX(($translateX)*(-1));}100% {opacity:1; -moz-transform:translateX(0);}}
+//   @keyframes fade-in-show{0% {opacity:0; transform:translateX(($translateX)*(-1));}100% {opacity:1; transform:translateX(0);}}
+//   animation-name:fade-in-show;
+// }
+// @mixin fade-in-show($translateX){
+//   animation-fill-mode:both;
+//   display: inline-block !important;
+//   @include fade-in-show-keyframe($translateX);
+// }
+
+.animaStyle-small-show{
+  display:inline-block !important;
+  @include fade-in(1.2s);
+}
+
+.animaStyle-medium-show{
+  display:inline-block !important;
+  @include fade-in(0.9s);
+}
+
+.animaStyle-large-show{
+  display:inline-block !important;
+  @include fade-in(0.7s);
+}
+
+@-webkit-keyframes wrapper-show { 0% {width: 0;} 100% {width: rem(133px);} }
+@-moz-keyframes    wrapper-show { 0% {width: 0;} 100% {width: rem(133px);} }
+@keyframes         wrapper-show { 0% {width: 0;} 100% {width: rem(133px);} }
+
+.animaStyle-wrapper-show{
+  display:inline-block !important;
+  animation-name: wrapper-show;
+  animation-duration: 0.4s;
+  animation-fill-mode:both;
+}
+
+// @mixin fade-in-hide-keyframe($translateX){
+//   @-webkit-keyframes fade-in-hide{0% {opacity:1; -webkit-transform:translateX(0);}100% {opacity:0; -webkit-transform:translateX(($translateX)*(-1)); visibility:hidden}}
+//   @-moz-keyframes fade-in-hide{0% {opacity:1; -moz-transform:translateX(0);}100% {opacity:0; -moz-transform:translateX(($translateX)*(-1)); visibility:hidden}}
+//   @keyframes fade-in-hide{0% {opacity:1; transform:translateX(0);}100% {opacity:0; transform:translateX(($translateX)*(-1)); visibility:hidden}}
+//   animation-name:fade-in-hide;
+// }
+
+// @mixin fade-in-hide($translateX){
+//   animation-fill-mode:both;
+//   display: inline-block !important;
+//   @include fade-in-hide-keyframe($translateX);
+// }
+
+@-webkit-keyframes local-fade-out { 0% {opacity: 1;} 100% {opacity: 0; visibility:hidden;} }
+@-moz-keyframes    local-fade-out { 0% {opacity: 1;} 100% {opacity: 0; visibility:hidden;} }
+@keyframes         local-fade-out { 0% {opacity: 1;} 100% {opacity: 0; visibility:hidden;} }
+
+.animaStyle-small-hide{
+  display:inline-block !important;
+  animation-name:local-fade-out;
+  @include animated(0.2s);
+}
+
+.animaStyle-medium-hide{
+  display:inline-block !important;
+  animation-name:local-fade-out;
+  @include animated(0.3s);
+}
+
+.animaStyle-large-hide{
+  display:inline-block !important;
+  animation-name:local-fade-out;
+  @include animated(0.4s);
+}
+
+@-webkit-keyframes wrapper-hide { 0% {width: rem(133px);} 100% {width: 0;} }
+@-moz-keyframes    wrapper-hide { 0% {width: rem(133px);} 100% {width: 0;} }
+@keyframes         wrapper-hide { 0% {width: rem(133px);} 100% {width: 0;} }
+
+.animaStyle-wrapper-hide{
+  display:inline-block !important;
+  animation-name: wrapper-hide;
+  @include animated(0.4s);
 }
```

Comment:
Could all these keyframes for animation be replaced by animation mixins we wrote in ````src/themes/animation.scss```?